app/services/container_dashboard_service.rb

```diff
@@ -0,0 +1,82 @@
+class ContainerDashboardService
+  def initialize(provider_id)
+    @provider_id = provider_id
+    @ems = ExtManagementSystem.find(@provider_id) unless @provider_id.nil?
```

Comment:
using ManageIQ::Providers::ContainerManager instead of ExtManagementSystem is more correct imo